File: curso POO uber/PHP/Car.php
<?php
    require_once('Account.php');
    class Car {
        public $id;
        public $license;
        public $driver;
        protected $passenger; // Es importante que esta propiedad esté marcada como protected

        public function __construct($license, $driver){
            $this->license = $license;
            $this->driver = $driver;
        }

        public function print_data_car(){
            echo "| License: $this->license,| Driver: {$this->driver->name}, | Document: {$this->driver->document}"."</br>";
        }
    }
?>

File: curso POO uber/Java/Car.java
public class Car {
    private Integer id;
    private String  license;
    private Account  driver;
    private Integer passenger;

    public Car(String  license, Account  driver  ){
        this.license = license;
        this.driver = driver;
    }

    void print_data_car(){
        System.out.println("License : " + license + " | Driver : " + driver.name+ " | Passengers: " + passenger);
    }

    public Integer getPassenger(){ 
        return passenger;
    }

    public void setPassenger(Integer passenger){
        if(passenger == 4){
            this.passenger = passenger;
        }else{
            System.out.println("You need to assign 4 passenger");
        } 
        
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getLicense() {
        return license;
    }

    public void setLicense(String license) {
        this.license = license;
    }

    public Account getDriver() {
        return driver;
    }

    public void setDriver(Account driver) {
        this.driver = driver;
    } 




    
}
